src/panels/lovelace/hui-editor.ts

```diff
@@ -112,11 +123,32 @@ class LovelaceFullConfigEditor extends hassLocalizeLitMixin(LitElement) {
           font-family: "Courier New", Courier, monospace;
           padding: 8px;
         }
+
+        .save-button {
+          opacity: 0;
+        }
+
+        .saved {
+          animation-name: opacityOn;
```

Comment:
Can't we do this without animations by just adding a `transition: opacity 1.5s` ?

Reply:
Oh right, with an animation it will fade out again?

Reply:
How did I forget about transition... I need to pack up my bags on being a web dev